docs/reference/compatibility.txt

```diff
@@ -89,12 +90,14 @@ is deprecated.
      - |checkmark|
      - |checkmark|
      - |checkmark|
+     - |checkmark|
      -
      -
      -
      - |checkmark|
 
    * - 7.3
+     - |checkmark| [#mongoid-7.3-ruby-3.0]_
```

Comment:
These footnotes aren't appropriate here because they reference 3.0 right?

Reply:
Actually, footnotes say "Mongoid version 7.x.y or higher is required.", and this is correct for both Ruby 3.0 and Ruby 3.1.

Reply:
Actually, footnotes say "Mongoid version 7.x.y or higher is required.", and this is correct for both Ruby 3.0 and Ruby 3.1.